debug/testlib.py

```diff
@@ -947,6 +948,32 @@ def __enter__(self):
     def __exit__(self, _type, _value, _traceback):
         self.gdb.pop_state()
 
+
+def load_excluded_tests(excluded_tests_file, target_name):
+    result = []
+    if excluded_tests_file is None or len(excluded_tests_file) == 0:
+        return result
+
+    target_excludes = {}
+    with open(excluded_tests_file) as file:
+        raw_data = yaml.safe_load(file)
+        for (target, test_list) in raw_data.items():
+            if not isinstance(test_list, list):
+                raise ValueError(f"Target \"{target}\" does not contain a test list", excluded_tests_file, test_list)
```

Comment:
```suggestion
                raise ValueError(f"Target {target!r} does not contain a test list", excluded_tests_file, test_list)
```

Reply:
This way python will add the quotes if it is a string, and also escape newlines or whatever else might end up in the string if the data is malformed.